dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -1193,7 +1195,7 @@ public User setUserPrivilege(@Parameter(hidden = true, name = "user")@Auth User
             + "", description = "A list of GitHub Events for the logged in user", content = @Content(array = @ArraySchema(schema = @Schema(implementation = LambdaEvent.class))))
     @ApiOperation(value = "See OpenApi for details")
     public List<LambdaEvent> getUserGitHubEvents(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User authUser,
-            @QueryParam("offset") Integer offset,
+            @NotNull @QueryParam("offset") Integer offset,
```

Comment:
We could probably also bound this with a `Min(0)` or similar.  Instead of `NotNull`, an option would be to specify a default value, that way we don't have to provide the offset if we're lazy.